### PR TITLE
3.1.0

### PR DESCRIPTION
**Pull request by tyler-sommer**
- [x] Sami api documentation generation plugin (#6)
- [x] Disabling remotes now also disables webhooks. Packages belonging to disabled remotes no longer appear in the Packages listing (#10)
- [ ] Documentation update

### CODE REVIEW

File: src/Migration/Version3100.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Migration script for 3.0->3.1
+ *
+ * Only necessary if upgrading from 3.0 to 3.1.
+ *
+ * @version 3.1.0
+ */
+class Version3100 extends AbstractMigration
```
Comment (Wirone) [inline src/Migration/Version3100.php]: Why did you change migrations' naming convention?

**Comment by bdandoy** [comment]
When is this going to be merged in?

**Comment by tyler-sommer** [comment]
@bdandoy I will shoot for having this merged, and 3.1 tagged, in the next two weeks.

File: src/Migration/Version3100.php (hunk 1 above)
Comment (tyler-sommer) [inline src/Migration/Version3100.php]: This convention is a holdover from when Packages was an internal Terramar Labs project. We deployed based on versions, and we always have one migration per version, so we opted to name the migrations after the version they represented. This convention is still compatible with doctrine-migrations, represented via: XY0Z where X is major, Y is minor, and 0Z is the patch level with 0 prepended if patch < 10.
